helm/testapp-chart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Load Test App
+name: testapp-chart
```

Comment:
I'm not sure about the name. I think we only want to use this for load testing and we already have our e2e-app.

https://github.com/giantswarm/e2e-app

What about `loadtest-app` to use the new naming style?

Reply:
Scratch that. I think we should keep things simple and use `appr`.

I still have a doubt on the naming. What about renaming the repo to `loadtest-app` and we make the chart `loadtest-app-chart`?

Reply:
yeah good idea